cluster/saltbase/salt/calico/calico-policy-agent.manifest

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: calico-policy-agent
+  namespace: calico-system
+  labels:
+    version: latest
+    projectcalico.org/app: "policy-agent"
+spec:
+  hostNetwork: true
+  containers:
+    - name: policyagent
+      image: calico/k8s-policy-agent:v0.1.3
```

Comment:
Does this support multi-master?

Might want to have a look here to add leader election if we so desire: https://github.com/projectcalico/k8s-policy/blob/master/examples/calico-policy-agent.yaml